Sources/TensorFlow/Core/Tensor.swift

```diff
@@ -553,6 +551,47 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
+//===------------------------------------------------------------------------------------------===//
+// Multiplicative Group
+//===------------------------------------------------------------------------------------------===//
+
+extension Tensor: PointwiseMultiplicative where Scalar: Numeric {
+    /// The scalar one tensor.
+    @inlinable
+    public static var one: Tensor { Tensor(1) }
+
+    /// Returns the pointwise reciprocal of `self`.
+    @inlinable
+    public var reciprocal: Tensor { 1 / self }
+
+    /// Multiplies two tensors and produces their product.
+    /// - Note: `*` supports broadcasting.
+    @inlinable
+    @differentiable(vjp: _vjpMultiply(lhs:rhs:) where Scalar: TensorFlowFloatingPoint)
+    public static func * (lhs: Tensor, rhs: Tensor) -> Tensor {
```

Comment:
This should be moved where it was, because it is not a protocol requirement and that `PointwiseMultiplicative` may undergo major changes.

Reply:
Done.